shopinvader_locomotive_reset_password/services/customer.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Akretion (http://www.akretion.com).
+# @author Sébastien BEAU <sebastien.beau@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.addons.component.core import Component
+from datetime import datetime
+
+
+class Customer(Component):
+    _inherit='shopinvader.customer.service'
+
+    def _assign_cart_and_get_store_cache(self):
+        res = super(Customer, self)._assign_cart_and_get_store_cache()
+        shop_partner = self.env['shopinvader.partner'].search([
+            ('backend_id', '=', self.shopinvader_backend.id),
+            ('record_id', '=', self.partner.id),
+            ])
+        if not shop_partner.date_initialisation:
+            shop_partner.date_initialisation = datetime.now()
```

Comment:
@sebastienbeau The `date_initialisation`should be the `create_date`of the binding. No?

Reply:
In fact it's not exactly the same.
In this case we have created the binding because the customer exist in the database and now we just want to know when the customer have initialized his password. It's mostly usefull in case of migration. You have a lot of customer and you want to follow who have updated his account

Reply:
Are you ok for initialization_date instead of ?